AAI/CHANGES_1.2.md

```diff
@@ -80,3 +80,15 @@ A short Elliptic Curve key of around 256 bits provides the same security as a 30
 For a detailed discussion of signing algorithms, see the article
 [JWTs: Which Signing Algorithm Should I Use?](https://www.scottbrady91.com/jose/jwts-which-signing-algorithm-should-i-use)
 
+## Media types for JWTs
+
+In version 1.0, all the mentioned JWTs (access tokens, visas) used in their `typ` (media type) header parameter
+the generic value `JWT` that marks a generic JWT.
+
+In version 1.2, the `typ` header parameter is used to distinguish the various types of JWTs:
+
+- access tokens conforming to [RFC9038](https://datatracker.ietf.org/doc/html/rfc9068#section-2.1)
+  use the value `at+jwt`
+- Passports use the value `vnd.ga4gh.passport+jwt`
```

Comment:
Do we need an allowance here for backwards compatible use of `JWT` for passports too - I know that 1.0 wasn't clear about signed passports as a concept - but because of that it is possible that there are implementations out there that make signed passports from the /userinfo endpoint and use an older JWT type?

Reply:
This is what we have under the "Conformance of Brokers" section in the `1.2-draft-main` branch:
> When presented with a valid access token, the endpoint MUST return claims in the specified [Format] (#claims-sent-to-data-holder-by-a-broker-via-token-or-userinfo) using either an application/json or application/jwt encoding.

I don't feel strongly, but since the UserInfo endpoint `application/jwt` response is defined in the OIDC spec I think we could omit it here.